apps/email/js/sync.js

```diff
@@ -311,17 +312,25 @@ define(function(require) {
             }
 
             if (model.getAccountCount() === 1) {
-              subject = mozL10n.get('new-emails-notify-one-account', {
-                n: count
-              });
+              subjectL10n = {
+                id: 'new-emails-notify-one-account',
+                args: { n: count }
+              };
             } else {
-              subject = mozL10n.get('new-emails-notify-multiple-accounts', {
-                n: count,
-                accountName: result.address
-              });
+              subjectL10n = {
+                id: 'new-emails-notify-multiple-accounts',
+                args: {
+                  n: count,
+                  accountName: result.address
+                }
+              };
             }
+            
+            bodyL10n = mozL10n.formatValue('senders-separation-sign')
+              .then(separator => {
+                return newFromNames.join(separator);
+            });
```

Comment:
Once the separator is known, and `newFromNames.join(separator)` is done, then I believe bodyL10n should be `{raw: newFromNames.join(separator)}` since it will already be localized.